src/components/routes/Host/ListingForm/ListingInfoForm/ListingInfoForm.tsx

```diff
@@ -41,6 +42,9 @@ const ListingInfoForm = (props: FormikProps<ListingInput>): JSX.Element => {
             <Svg className="suffix" src="utils/carat-down" />
           </label>
         </SelectBoxWrapper>
+        <ErrorMessageWrapper>
```

Comment:
for the errors and inputs, do you think there's a way to loop through and dynamically create these?

Reply:
cool thanks! updated